### DISCUSSION TITLE
Start Record in the Logs

### DISCUSSION THREAD

**Discussion by Bag72** [alexjustesen/speedtest-tracker#1329] [Q&A]
I'm getting regular failures on the speedtests.    It appears to create a log entry for the start of the test and when it is successful it deletes that and adds in the record with the stats.   If it fails it leaves the start record there.  As it has no useful information in there what is the point of it when we have a failure record?

**Comment by alexjustesen** [comment]
Going to tie this into #1260 as I'm trying to capture more error messages from the Speedtest CLI along with states of running, failed and completed tests.
